{"uuid":"21d1c5d6-6c92-4663-a22e-9a69c0f0df80","historyId":"e5ca53f41d059a5524dd6bbb18898fa8","fullName":"com.thetestingacademy.tests.vwo.pageObjectModelTC.TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM.testFail","labels":[{"name":"package","value":"com.thetestingacademy.tests.vwo.pageObjectModelTC.TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM"},{"name":"testClass","value":"com.thetestingacademy.tests.vwo.pageObjectModelTC.TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM"},{"name":"testMethod","value":"testFail"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"com.thetestingacademy.tests.vwo.pageObjectModelTC.TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM"},{"name":"host","value":"DESKTOP-CJN9AFV"},{"name":"thread","value":"13232@DESKTOP-CJN9AFV.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Mangesh"}],"links":[],"name":"testFail","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat org.testng.Assert.assertTrue(Assert.java:67)\r\n\tat com.thetestingacademy.tests.vwo.pageObjectModelTC.TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM.testFail(TestVWOLogin_05_TakeScreen_Retry_Prop_Improved_POM.java:45)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n"},"stage":"finished","description":"Verify that invalid creds give error message","steps":[],"attachments":[{"name":"Log output","source":"dd934092-2549-4c2c-bf0e-cfa843bb3389-attachment.txt","type":"text/plain"}],"parameters":[],"start":1759588551913,"stop":1759588560457}